rosdep/python.yaml

```
@@ -7615,6 +7612,12 @@ python3-psutil:
   rhel: ['python%{python3_pkgversion}-psutil']
   slackware: [psutil]
   ubuntu: [python3-psutil]
+python3-psycopg2:
+  debian: [python3-psycopg2]
+  fedora: [python3-psycopg2]  
```

Remove trailing whitespace:

```suggestion
  fedora: [python3-psycopg2]
```